pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	utilversion "k8s.io/apimachinery/pkg/util/version"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// DRAPluginName is the name of the in-tree DRA Plugin.
+	DRAPluginName = "kubernetes.io/dra"
```

Comment:
What does this mean? What "in-tree DRA plugin" are you referring to?

Reply:
This is how I called plugin code in the pkg/kubelet/cm/dra/plugin/plugin.go
If you think it's confusing - let me know how to better call it.

Reply:
I actually think we can just get rid of this, I don't see much value in having this `log()` wrapper (which is the only place this is used).